assets/src/vo/date-time/datetime.js

```diff
@@ -0,0 +1,1156 @@
+	 * @param {...DateTime} datetimes
+	 * @return {DateTime} A new DateTime representing the earliest point in
+	 * time.
+	 */
+	static min( ...datetimes ) {
+		return new DateTime(
+			moment.min(
+				DateTime[ privateMethods.extractMomentsFromDateTimes ](
+					...datetimes
+				)
+			)
+		);
+	}
+
+	/**
+	 * Constructs a DateTime from an ISO 8601 string.
+	 *
+	 * @param {string} ISOString
+	 * @param {string} timezone
+	 * @param {string} locale
+	 * @return {DateTime} An instance of DateTime
+	 */
+	static fromISO(
+		ISOString,
+		timezone = DEFAULT_TIMEZONE_STRING,
+		locale = DEFAULT_VALID_LOCALE
+	) {
+		return new DateTime( ISOString, timezone, locale );
+	}
+
+	/**
+	 * Constructs a DateTime from an ISO 8601 String
+	 * Differs with `fromISO` in that this allows passing a offset value
+	 * instead of a timezone string.
+	 *
+	 * @param {string} ISOString
+	 * @param {number} offset  In minutes unless > -16 or < -16 in which case it
+	 * is treated as hours.
+	 * @param {string} locale
+	 * @return {DateTime}  An instance of DateTime
+	 */
+	static fromISOWithOffset(
+		ISOString,
+		offset = DEFAULT_OFFSET,
+		locale = DEFAULT_VALID_LOCALE
+	) {
+		DateTime.assertISO8601IsValid( ISOString );
+		DateTime.assertIsOffset( offset );
+		DateTime.assertLocaleIsValid( locale );
+		const datetime = isEmpty( ISOString ) ?
+			moment().utcOffset( offset, true ).locale( locale ) :
+			moment.utc( ISOString )
+				.utcOffset( offset, true )
+				.locale( locale );
+		return new DateTime( datetime );
+	}
+
+	/**
+	 * Constructs a DateTime from a javascript Date object.
+	 *
+	 * @param {Date} date
+	 * @param {string} timezone
+	 * @param {string} locale
+	 * @return {DateTime} Returns an instance of DateTime
+	 */
+	static fromJSDate(
+		date,
+		timezone = DEFAULT_TIMEZONE_STRING,
+		locale = DEFAULT_VALID_LOCALE
+	) {
+		DateTime.assertIsDate( date );
+		DateTime.assertTimezoneIsValid( timezone );
+		DateTime.assertLocaleIsValid( locale );
+		return new DateTime(
+			moment( date ).tz( timezone ).locale( locale )
+		);
+	}
+
+	/**
+	 * Constructs a Datetime from a javascript Date object.
+	 *
+	 * The difference between this and fromJSDate is that this can be set with
+	 * an offset vs a timezone string.
+	 *
+	 * @param {Date} date
+	 * @param {number} offset
+	 * @param {string} locale
+	 * @return {DateTime} Returns an instance of DateTime
+	 */
+	static fromJSDateWithOffset(
+		date,
+		offset = DEFAULT_OFFSET,
+		locale = DEFAULT_VALID_LOCALE
+	) {
+		DateTime.assertIsDate( date );
+		DateTime.assertIsOffset( offset );
+		DateTime.assertLocaleIsValid( locale );
+		return new DateTime(
+			moment( date ).utcOffset( offset ).locale( locale )
+		);
+	}
+
+	/**
+	 * Constructs a DateTime (in utc) with milliseconds from epoch.
+	 *
+	 * @param {number} milliseconds
+	 * @param {string} locale
+	 * @return {DateTime} Returns an instance of DateTime
+	 * @throws TypeError
+	 */
+	static fromMillis( milliseconds, locale = DEFAULT_VALID_LOCALE ) {
```

Comment:
can we use the full word plz?

ie:

```js
static fromMilliseconds( milliseconds, locale = DEFAULT_VALID_LOCALE ) {
```

Reply:
From the description on the pull:

> For the most part I've tried to use an api that is similar in behaviour to the moment Luxon library.

https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#static-method-fromMillis

So... that's the intent behind using `fromMillis`.  If you absolutely must have the change I'll do it, but I don't think its necessary.